src/control/run_go_tests.sh

```diff
@@ -44,10 +44,16 @@ function setup_environment()
 
 	source "${build_source}"
 
-	LD_LIBRARY_PATH="${SL_PREFIX}/lib:${SL_SPDK_PREFIX}/lib:${LD_LIBRARY_PATH}"
-	export LD_LIBRARY_PATH
-	export CGO_LDFLAGS="-L${SL_SPDK_PREFIX}/lib -L${SL_PREFIX}/lib"
-	export CGO_CFLAGS="-I${SL_SPDK_PREFIX}/include"
+	# ugh, appease the linter...
+	LD_LIBRARY_PATH=${SL_PREFIX}/lib
+	LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${SL_SPDK_PREFIX}/lib"
+	LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${SL_HWLOC_PREFIX}/lib"
+	CGO_LDFLAGS=-L${SL_PREFIX}/lib
+	CGO_LDFLAGS="$CGO_LDFLAGS -L${SL_SPDK_PREFIX}/lib"
```

Comment:
```suggestion
	CGO_LDFLAGS+=" -L${SL_SPDK_PREFIX}/lib"
```
FWIW

Reply:
ah, right. thanks. sigh, i've forgotten how to write shell scripts...